docs/source/install/index.rst

```diff
@@ -3,15 +3,15 @@ Installation
 
 For a quick evaluation of |st2| and package-based installation, start from
 :doc:`Overview <./overview>`, and proceed with the guide for your Linux distribution.
-
+Please note that only 64 bit architecture is currently supported.
 
 .. rubric:: Installations
 
 .. toctree::
     :maxdepth: 1
 
     Reference Deployment <overview>
-    Ubuntu / Debian <deb>
+    Ubuntu / Debian 14.04 (Trusty) <deb>
```

Comment:
purely semantically, what debian is doing in this line at all? There's surely no `debian 14.04` in the wild. I'd rather switch the order.